code/modules/clothing/suits/armor.dm

```diff
@@ -391,16 +398,18 @@
 	desc = "A standard armor vest fielded for SolGov's infantry."
 	icon_state = "armor_solgov"
 	item_state = "armor_solgov"
+	armor = list("melee" = 15, "bullet" = 60, "laser" = 10, "energy" = 10, "bomb" = 40, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50)
 
 /obj/item/clothing/suit/armor/vest/solgov/Initialize()
 	. = ..()
-	allowed |= list(/obj/item/gun/energy/laser/terra, /obj/item/gun/energy/pulse/terra, /obj/item/tank)
+	allowed |= list(/obj/item/gun/ballistic/automatic/assualt/swiss_cheese, /obj/item/tank)
 
 /obj/item/clothing/suit/armor/vest/solgov/rep
 	name = "\improper SolGov Inspector armor vest"
 	desc = "A type I armor vest emblazoned with the SolGov logo."
 	icon_state = "armor_alt_solgov"
 	item_state = "armor_alt_solgov"
+	armor = list("melee" = 35, "bullet" = 30, "laser" = 30, "energy" = 40, "bomb" = 25, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50)
```

Comment:
if you follow up on the comment for the regular solgov armor please remove this line

Reply:
disregard